webpack/webpack.dev.cjs

```diff
@@ -39,8 +39,13 @@ module.exports = mergeWithRules({
       options: 'replace',
     },
   },
+  cache: 'replace',
 })(baseConfig, {
   mode: 'development',
   devtool: 'source-map',
+  cache: {
+    type: 'memory',
+    name: 'cactbot',
```

Comment:
```suggestion
-    name: 'cactbot',
```

`cache.name` only works with `type: 'filesystem'`

Reply:
Wait?
In development mode, the `cache` property is set to `memory` by default, isn't it?
So I can delete this block entirely?